package.json

```diff
@@ -27,6 +27,7 @@
     "babel-register": "^6.9.0",
     "compression-webpack-plugin": "~1.1.11",
     "coveralls": "^3.0.0",
+    "cross-env": "^5.2.0",
```

Comment:
@ekohl - I assume this can be added to the packaging whitelist?

Reply:
Yes it can since it's only needed for storybooks, which we ignore.